app/src/main/java/com/getsafetee/AppInitializer.java

```diff
@@ -1,7 +1,9 @@
-package com.getsafetee.safetee;
+package com.getsafetee;
```

Comment:
Do you have a rationale behind changing it from com.getsafetee.safetee to com.getsafetee?

Since "getsafetee" is the domain name and "safetee" is the app name